CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.23] - 2020-06-12
```

Comment:
Yeah again this is for when you are releasing the latest client. You add this date when it's about to be merge with master which Yas or I will do